parm/soca/berror/soca_setcorscales.yaml

```diff
@@ -0,0 +1,44 @@
+resolution:
+  mom6_input_nml: mom_input.nml
+  fields metadata: ./fields_metadata.yaml
+
+date: 0001-01-01T00:00:00Z
```

Comment:
If this runs every cycle, why not set the date appropriately here (and in the previous file?)

Reply:
It's done once, just not reflected yet in the scripts. I'll add switches in the future.